buildspec.yml

```diff
@@ -0,0 +1,33 @@
+version: 0.2
+
+env:
+  parameter-store:
+    AWS_ACCOUNT_ID: account_id
```

Comment:
How does this work? Is it coming from github secrets? If not that's an easy way to get this.